libraries/src/MVC/Model/Exception/ModelExceptionInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\CMS\MVC\Model\Exception;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('JPATH_PLATFORM') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Interface that all exceptions stemming from the model should implement for processing by the controller
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+interface ModelExceptionInterface
```

Comment:
```suggestion
interface ModelExceptionInterface extends \Throwable
```
See discussion https://github.com/joomla/joomla-cms/pull/38949/files#r997435617